src/richie/apps/core/templates/richie/icons.html

```diff
@@ -147,5 +147,9 @@
     <symbol id='icon-school' fill='currentColor' viewBox='0 0 24 24'>
       <path d='M4.86093507,12.7064857 L4.86093507,15.3820842 C4.86093507,16.0771685 5.24180318,16.7246443 5.85119216,17.0579039 L10.6120435,19.6573287 C11.1833457,19.9715449 11.8689083,19.9715449 12.4402105,19.6573287 L17.2010618,17.0579039 C17.8104508,16.7246443 18.1913189,16.0771685 18.1913189,15.3820842 L18.1913189,12.7064857 L12.4402105,15.8486476 C11.8689083,16.1628638 11.1833457,16.1628638 10.6120435,15.8486476 L4.86093507,12.7064857 Z M10.6120435,3.50852087 L2.58524812,7.88850413 C1.92825063,8.25032883 1.92825063,9.2024991 2.58524812,9.56432381 L10.6120435,13.9443071 C11.1833457,14.2585233 11.8689083,14.2585233 12.4402105,13.9443071 L20.0956595,9.76427956 L20.0956595,15.3916059 C20.0956595,15.9152995 20.5241361,16.3437762 21.0478297,16.3437762 C21.5715234,16.3437762 22,15.9152995 22,15.3916059 L22,9.28819443 C22,8.93589143 21.8095659,8.62167524 21.5048715,8.45028459 L12.4402105,3.50852087 C11.8689083,3.20382638 11.1833457,3.20382638 10.6120435,3.50852087 Z' id='Shape'></path>
     </symbol>
+
+    <symbol id='icon-cog' fill='currentColor' viewBox='0 0 24 24'>
```

Comment:
A possibility would have been to add a dedicated commit to icons addition, which can be independent 😊

Reply:
I had multiple commit that i squash on last rebase in order to make it easier.
I can extract this icon, setting link and settings link's story in a dedicated commit.